packages/number-input/README.md

```diff
@@ -62,28 +62,28 @@ or
-| `disabled`        | `boolean`                                              | Disables the component.                                                                                                                                                                                                                          | `false`   |
-| `placeholder`     | `string`                                               | The placeholder text shown in the input field before the user begins typing.                                                                                                                                                                     |           |
-| `size`            | `'xsmall'` \| `'small'` \| `'default'`                 | Determines the size of the input.                                                                                                                                                                                                                | `default` |
-| `state`           | `'none'`\| `'error'`                                   | Describes the state of the TextInput element before and after the input has been validated                                                                                                                                                       | `'none'`  |
-| `errorMessage`    | `string`                                               | Error message that appears below the input. Renders only if `state='error'`.                                                                                                                                                                     | `'none'`  |
-| `unit`            | `string`                                               | The string unit that appears to the right of the input if using a single unit. </br> </br>Required if using `unitOptions`. When using `unitOptions` this value becomes the controlled value of the select input.                                 | `default` |
-| `unitOptions`     | `Array<{displayName: string; value: string}>`          | The options that appear in the select element attached to the right of the input.                                                                                                                                                                | `default` |
+| Prop              | Type                                                   | Description                                                                                                                                                                                                                                      | Default                             |
```

Comment:
Is there a way to use our tsdocs to generate this table for us?

Reply:
yeah agreed, it would be nice to rely on the generated props table that exists on .design so we don't need to maintain this readme. I'm not clear on what the level of effort looks like to do so but would be happy to look further into this if we can allocate time to do so for all packages